### PR TITLE
Release 1.14.0

### PR DESCRIPTION
**Pull request by andyksaw**
## Overview of Changes
* Split login and SSO (https://github.com/andyksaw/ProjectCityBuild/pull/163)
* Add Stripe error handling (https://github.com/andyksaw/ProjectCityBuild/pull/181)
* AccountSettingsController clean up (https://github.com/andyksaw/ProjectCityBuild/pull/146)
* PasswordRecovery clean up (https://github.com/andyksaw/ProjectCityBuild/pull/182)
* Fixes deployment (https://github.com/andyksaw/ProjectCityBuild/pull/186)

### Deployment Steps (Optional)
* Change SSO endpoint to `/sso/discourse`
* Add env var `DISCOURSE_SSO_ENDPOINT="https://forums.projectcitybuild.com/session/sso"`

### Dependencies (Optional)
None

### Screenshots (Optional)
None

### CODE REVIEW

File: src/app/Http/Actions/AccountRegistration/ActivateUnverifiedAccount.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            throw new \Exception('This unactivated account does not exist');
+        }
+
+        $accountByEmail = $this->accountRepository->getByEmail($email);
+        if ($accountByEmail) {
+            // Account already activated
+            abort(410);
+        }
+
+        DB::beginTransaction();
```
Comment (freddyheppell) [inline src/app/Http/Actions/AccountRegistration/ActivateUnverifiedAccount.php]: Is there a reason for this versus the [closure method](https://laravel.com/docs/5.8/database#database-transactions), which essentially does the same thing?

File: src/app/Http/Middleware/Authenticate.php [hunk 2]
```diff
@@ -15,6 +15,8 @@ class Authenticate extends Middleware
     protected function redirectTo($request)
     {
         if (! $request->expectsJson()) {
+
```
Comment (freddyheppell) [inline src/app/Http/Middleware/Authenticate.php]: Whoops, I added this whitespace implementing my own redirect back to original request before I realised laravel already does that

File: src/resources/views/front/pages/donate/donate-error.blade.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+@section('title', 'Donation Error')
+@section('description', "")
+
+@section('contents')
+
+    <div class="card">
+        <div class="card__body card__body--padded">
+            <h1>There was a problem processing your donation</h1>
+
+            <p>{{ $message }}</p>
```
Comment (freddyheppell) [inline src/resources/views/front/pages/donate/donate-error.blade.php]: This isn't a great approach, ideally we'd redirect back to the original page with the amount stil filled in etc but considering that Checkout catches _most_ errors, and when we switch to new checkout it won't be an issue at all this should be fine for now

File: src/app/Http/Actions/AccountRegistration/ActivateUnverifiedAccount.php (hunk 1 above)
Comment (andyksaw) [reply src/app/Http/Actions/AccountRegistration/ActivateUnverifiedAccount.php]: Using the closure will consume the exception and fail silently. If an error we didn't expect happens, it's better that it fails and we get notified via Sentry

File: src/resources/views/front/pages/donate/donate-error.blade.php (hunk 3 above)
Comment (andyksaw) [reply src/resources/views/front/pages/donate/donate-error.blade.php]: For now I think this is fine. At least it's better than what we currently have, which is to throw a 500 error when the card is declined

File: src/app/Http/Actions/AccountRegistration/ActivateUnverifiedAccount.php (hunk 1 above)
Comment (freddyheppell) [reply src/app/Http/Actions/AccountRegistration/ActivateUnverifiedAccount.php]: Ah right, I would have thought there'd have been some way of getting Sentry to still report it but some quick googling didn't find anything

File: src/resources/views/front/pages/donate/donate-error.blade.php (hunk 3 above)
Comment (freddyheppell) [reply src/resources/views/front/pages/donate/donate-error.blade.php]: Yeah, and anyway when me migrate to new checkout it'll be irrelevant anyway as Stripe actually perform the charge on their end

**Pull request merged**